web-console/1-homepage/catalog.md

```diff
@@ -36,9 +33,32 @@ On smaller viewport sizes, the catalog tabs should stack. ![hover](img/OpenShift
 - For secondary levels of categorization, there should again be an `Other` category if there are items to display (not shown in mockup).
 - Clicking on a subcategory will open a scrollable panel that contains the catalog items.
 
+### Responsive State
+![mobile-1](img/OpenShift-Next-Mobile-V2-scrolledup.png)
+
+- On smaller viewport sizes, the catalog tabs should stack with arrows to the right.
+- Deploy Image and Import YAML should be displayed below all primary categories and should have a different visual treatment from the tabs.
+
+![mobile-2](img/OpenShift-Next-Mobile-subcategories.png)
+
+- If subcategories exist, selecting the primary category will bring users to another screen to view subcategories as tiles.
+- Clicking back will bring users back to the home page with all primary tabs listed.
+
+![mobile-3](img/OpenShift-Next-Mobile-subcategoryselected-V2.png)
+
+- Selecting a subcategory tile will bring users to another screen to view catalog items by scrolling.
+
+![mobile-4](img/OpenShift-Next-Mobile-subcategorydropdown.png)
+- The category title will be listed with a dropdown option to switch to another subcategory without navigating back to tiles.
+- The current selection will be highlighted with the icon in full color. All other images are muted.
```

Comment:
Ok.  We typically still code it even though it's not used since Bootstrap/PatternFly have hover states at mobile.